xray/src/bin/build_xray_quadtree.rs

```diff
@@ -10,7 +10,7 @@ extern crate quadtree;
 extern crate scoped_pool;
```

Comment:
Grep for "extern crate" and remove them? Same for macro_use.

Reply:
Nice, done!